### DISCUSSION TITLE
How to change the symbol of Checkbox widget?

### DISCUSSION THREAD

**Discussion by Tim-Cao** [Textualize/textual#3173] [Q&A] [answered]
The default symbol of checkbox is cross symbol not a check symbol.
![image](https://github.com/Textualize/textual/assets/52661397/016a6de6-ce06-45ef-a434-f8dcaefc3d9a)
It makes us confused should we check or not?

For example,
![image](https://github.com/Textualize/textual/assets/52661397/50daa8dc-63d8-4bdf-b54c-6f237fc0cffb)

**Comment by davep** [comment, marked as answer]
We evaluated check symbols but none were stable in their display across various operating systems and terminals.

**Reply by Tim-Cao** [reply]
Is it allowed to customized by css?

**Reply by davep** [reply]
No, it's set in the code of the widget.

**Reply by Tim-Cao** [reply]
I would like to use RadioButton to instead, however, the radio buton is misaligned, is it a bug?
![image](https://github.com/Textualize/textual/assets/52661397/66b3246d-7c8e-4b0d-800b-f89fd33ac522)

**Reply by TomJGooding** [reply]
> I would like to use RadioButton to instead, however, the radio buton is misaligned, is it a bug?

No, this is a font issue and not a bug with Textual. See #3091.

**Reply by Tim-Cao** [reply]
I see. Thanks. I use the switch to instead of checkbox.
![image](https://github.com/Textualize/textual/assets/52661397/0f8ce4c3-c6a6-4c77-9b03-5c8bbdc6610a)
